src/agent/subagents/linux/drbd.cpp

```diff
@@ -183,7 +183,7 @@ LONG H_DRBDDeviceList(const TCHAR *pszCmd, const TCHAR *pArg, StringList *pValue
 	{
 		if (s_devices[i].id != -1)
 		{
-			_sntprintf(szBuffer, 1024, _T("/dev/drbd%d %hs %hs/%hs %hs/%hs %hc"),
+                        _sntprintf(szBuffer, 1024, _T("/dev/drbd%d %hs %hs/%hs %hs/%hs %d"),
```

Comment:
DRBD protocol indicated with single character, so %hc is correct  format here. %d will print that character's code instead.

Reply:
May be revert code to %hc and cast s_devices[i].protocol to proper type?

Reply:
Yes, actually it could be %c and cast to TCHAR to avoid compiler warnings.